### DISCUSSION TITLE
Is there an option to group multiple processes of programs?

### DISCUSSION THREAD

**Discussion by bege10** [winsiderss/systeminformer#1940] [Questions / Answers] [answered]
I miss the view which the Windows Task Manager shows that multiple processes of a program (e.g. Firefox) are grouped together and can be (un-)collapsed. Is there an option to do so in SystemInformer or should I create a feature request?

**Comment by jxy-s** [comment]
You can change the column sort by clicking it. There is also a "collapse by default" option.

**Reply by bege10** [reply]
Thanks. I don't find that option. Where is it?

**Reply by jxy-s** [reply, marked as answer]
Right click the process.

Could add a general option if there isn't one to collapse "all" on startup. I didn't find any in the advanced options by looking at the source.

Won't be back to look closer until next week. Feel free to create a feature request if it's not to your liking.

**Reply by bege10** [reply]
I see, thank you. That only works in tree view, not if one chooses sorting by name or by CPU usage. I'll create a feature request.
